README.md

```diff
@@ -10,6 +10,9 @@ It does this by adding traditional database-backed audits for durable jobs enque
 When an audited job isn't marked as complete after a certain amount of time (10 minutes by default),
 it's considered failed, and will be re-enqueued with exponential backoff until it succeeds or is removed.
 
+Resque/Durable allows to re-enqueue the job immediately by requesting it. An audited job will not be marked as
+complete and the job will be re-enqueued immediately by a background durable monitor.
+
```

Comment:
Would suggest moving this under `Usage`, and amending as follows:

```

## Re-enqueuing gracefully

To re-enqueue the job gracefully, and without waiting for the the audit failure timeout, call the `re_enqueue_immediately!` class method at any point while performing the job (usually at or close to the end). 

Resque/Durable will not mark the job as complete. Instead, it will mark the job as failed and reset the exponential backoff. The background durable monitor will then re-enqueue the job as described above.

A common use case for this would be to gracefully stop a long-running job for a worker restart, and retry the job as soon as possible after the worker restart.

```